bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java

```diff
@@ -356,8 +356,8 @@ public ResultScanner getScanner(byte[] family, byte[] qualifier) throws IOExcept
   @Override
   public void put(Put put) throws IOException {
     LOG.trace("put(Put)");
-    RowMutation request = hbaseAdapter.adapt(put);
-    mutateRow(put, request, "put");
+    RowMutation rowMutation = hbaseAdapter.adapt(put);
```

Comment:
Please make this change in a separate PR.  There's no reason to lump this change with `mutateRows`